torch_xla/csrc/ops/ops.cpp

```diff
@@ -882,6 +882,27 @@ NodePtr NanToNum(const Value& input, const Value& nan, const Value& posinf,
                    input.shape(), std::move(lower_fn));
 }
 
+NodePtr ExpandAsDynamicShapes(const Value& static_input,
```

Comment:
`DynamicExpand` might be a better name since we also have `DynamicReshape` in [here](https://github.com/pytorch/xla/blob/bbd87ad515744e52a4a63eafcd14fb3339a43724/torch_xla/csrc/helpers.cpp#L348). In the future we should just make `expand` support dynamic shape but that would require a frontend API change

Reply:
I agree and it makes sense.